content/docs/getting_started/traffic_split.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "OSM manual demo"
+description: "The manual demo is a step-by-step walkthrough set of instruction of the automated demo."
+type: docs
+weight: 4
+aliases: = ["/docs/install/manual_demo/"]
+---
+
+### Configure Traffic Split between Two Services
+
+We will now demonstrate how to balance traffic between two Kubernetes services, commonly known as a traffic split. We will be splitting the traffic directed to the root `bookstore` service between the backend services `bookstore-v1` and `bookstore-v2`.
+
+### Deploy bookstore v2 application
+
+To demonstrate usage of SMI traffic access and split policies, we will now deploy version v2 of the bookstore application (`bookstore-v2`) - remember that if you are using openshift, you must add the security context constraint to the bookstore-v2 service account as specified in the [installation guide](/docs/install/#openshift). A `bookstore` service will also be created to serve as the root of our traffic split between `bookstore-v1` and `bookstore-v2`.
+
+```bash
+# Contains the bookstore-v2 Kubernetes Service, Service Account, Deployment and SMI Traffic Target resource to allow
```

Comment:
The bookstore service is also contained in `bookstore-v2.yaml`. Could this be moved to a separate yaml?